src/main/java/fi/otavanopisto/kuntaapi/server/integrations/tpt/client/model/DocsEntry.java

```diff
@@ -0,0 +1,453 @@
+package fi.otavanopisto.kuntaapi.server.integrations.tpt.client.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+import java.util.*;
+
+public class DocsEntry implements Serializable {
+
+  private static final long serialVersionUID = -4442994963182715417L;
+
+  @JsonProperty("hakuTyosuhdetyyppikoodi")
+  private Integer hakuTyosuhdetyyppikoodi;
+
+  @JsonProperty("ammatti")
+  private List<String> ammatti;
+
+  @JsonProperty("tyoaikatekstiYhdistetty")
+  private String tyoaikatekstiYhdistetty;
+
+  @JsonProperty("hakuPaattyy")
+  private String hakuPaattyy;
+
+  @JsonProperty("maa")
+  private String maa;
+
+  @JsonProperty("asuntomahdollisuus")
+  private Boolean asuntomahdollisuus;
+
+  @JsonProperty("vuokratyopaikka")
+  private Boolean vuokratyopaikka;
+
+  @JsonProperty("etatyo")
+  private Boolean etatyo;
+
+  @JsonProperty("kieli")
+  private String kieli;
+
+  @JsonProperty("anonyymiTyonantaja")
+  private Boolean anonyymiTyonantaja;
+
+  @JsonProperty("tehtavanimi")
+  private String tehtavanimi;
+
+  @JsonProperty("ammattiLevel3")
+  private String ammattiLevel3;
+
+  @JsonProperty("tyonKestoKoodi")
+  private Integer tyonKestoKoodi;
+
+  @JsonProperty("kuvausteksti")
+  private String kuvausteksti;
+
+  @JsonProperty("englanti")
+  private Boolean englanti;
+
+  @JsonProperty("sijainti")
+  private String sijainti;
+
+  @JsonProperty("ilmoituspaivamaara")
+  private OffsetDateTime ilmoituspaivamaara;
+
+  @JsonProperty("tyonKesto")
+  private String tyonKesto;
+
+  @JsonProperty("tyonKestoTekstiYhdistetty")
+  private String tyonKestoTekstiYhdistetty;
+
+  @JsonProperty("id")
+  private String id;
+
+  @JsonProperty("hakuTyoaikakoodi")
+  private Integer hakuTyoaikakoodi;
+
+  @JsonProperty("kesatyopaikka")
+  private Boolean kesatyopaikka;
+
+  @JsonProperty("ilmoitettuTyomarkkinatLehdessa")
+  private Boolean ilmoitettuTyomarkkinatLehdessa;
+
+  @JsonProperty("timestamp")
+  private Long timestamp;
+
+  @JsonProperty("tyoaika")
+  private String tyoaika;
+
+  @JsonProperty("itsepalvelu")
+  private Boolean itsepalvelu;
+
+  @JsonProperty("vahtipostitus")
+  private Boolean vahtipostitus;
+
+  @JsonProperty("hakemusOsoitetaan")
+  private String hakemusOsoitetaan;
+
+  @JsonProperty("paikkalukumaara")
+  private Integer paikkalukumaara;
+
+  @JsonProperty("kunta")
+  private String kunta;
+
+  @JsonProperty("tyonantajanNimi")
+  private String tyonantajanNimi;
+
+  @JsonProperty("euresTyopaikka")
+  private Boolean euresTyopaikka;
+
+  @JsonProperty("ytunnus")
+  private String ytunnus;
+
+  @JsonProperty("mainAmmattikoodi")
+  private Integer mainAmmattikoodi;
+
+  @JsonProperty("oppisopimus")
+  private Boolean oppisopimus;
+
+  @JsonProperty("_version_")
+  private Long _version_;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "_version_" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)